WebApp/SharePanel.ascx

```diff
@@ -45,7 +45,7 @@ limitations under the License.
   </div>
   <div id="pnlPrint" class="share">
     <div class="share_box">
-      <span class="seg_title">Create a Printable PDF Map</span>
+      <span class="FunctionLabel">Create a Printable PDF Map</span>
```

Comment:
```seg_title``` to ```FunctionLabel``` ??

Reply:
seg_title change to Functionalabel in this file (SharePanel) .  Functionalabel  class used for change color and font size